README.md

```diff
@@ -121,11 +121,24 @@ The source JavaScript is written in ES6 (located in the `src/js` directory) and
 To version bump the repo and prepare a release, run `npm version major|minor|patch` (following [semver](https://semver.org/)), then (something like) `git push origin HEAD --tags`. It'll automatically update the version number where it appears.
 
 ## Tests
+We would like to get to 100% test coverage and are tracking our progress in this project: [BookReader Fidelity](https://github.com/internetarchive/bookreader/projects/5)
 
-Tests use [QUnit](https://qunitjs.com/). To run all tests, start up an HTTP server at the root of the project and navigate to `/tests/qunit.html` in a browser.
+### End to end tests
+We also have end to end tests using [Testcafe](https://devexpress.github.io/testcafe/documentation/getting-started/).  We write tests for the repo itself and also for our use on archive.org. You can read about them in [here](./tests/e2e/README.md). These are relatively easy to do, and a fantastic way of getting introduced to the wonders of BookReader.  Check the project board for open tickets to work on.  And if you don't see a test for something you spotted, feel free to make an issue.
+
+### Unit tests
+We have unit tests and use Jest to run them.
+For mocks, we use Jest's internal mocking mechanism and Sinon to set spies.
+
+### QUnit tests
```

Comment:
Axe this section; the qunit tests aren't part of ci, so likely out-of-date. I think jest+testcafe are good enough. Let's consider qunit deprecated.

Reply:
does this mean we delete the qunit tests, too?

Reply:
I think we should, but don't think we need that right now (your call).

Reply:
A new issue might be good 👍

Reply:
ok sounds good, will make a new issue

Reply:
here's the issue: https://github.com/internetarchive/bookreader/issues/272

Reply:
Oh, let's still remove the readme section here. We don't want people opening qunit PRS, so this section is misleading.

Reply:
done